hi-ml-cpath/src/SSL/configs/HistoSimCLRContainer.py

```diff
@@ -51,14 +51,15 @@ def _get_transforms(self, augmentation_config: Optional[CfgNode],
 
     @staticmethod
     def get_transforms(apply_augmentations: bool) -> ImageTransformationPipeline:
-        transforms: List[Any] = []
+        transforms: List[Any] = [StainNormalization()]         # Common transforms
```

Comment:
do we want to make this change at this level or rather override it in specific configs? 
What about adding a function get_preprocessing_tranforms that returns an empty list by default which we can override in the container, I don't think StainNormalization was part of SimCLR augmentations. 
You can also add a function get_augmentations_transforms() to replace lines 58-62 so that we can flexibly change in sub classed containers without too much duplications

Reply:
I have now made the suggested change, thank you!